src/test/java/com/spring/boot/util/CustomFileReaderImplTest.java

```diff
@@ -0,0 +1,28 @@
+package com.spring.boot.util;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CustomFileReaderImplTest {
```

Comment:
what about valid case, don't see tests for it

Reply:
done